test/components/NavBar/OidcLoginComponent.test.jsx

```diff
@@ -0,0 +1,32 @@
+import { render, cleanup } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { expect, it, vi, afterEach } from 'vitest';
+import OidcLoginComponent from '../../../src/components/NavBar/OidcLoginComponent';
+
+afterEach(() => {
+  vi.clearAllMocks();
+  cleanup();
+});
+
+it('renders correctly', () => {
+  const { container } = render(<OidcLoginComponent />);
+  expect(container).toMatchSnapshot();
+});
+
+it('sets OIDC provider on login', async () => {
+  vi.mock('@inrupt/solid-ui-react', () => ({
+    LoginButton: ({ children }) => <div>{children}</div>
```

Comment:
Mock the Inrupt Login button so it doesn't try to fetch a session that doesn't exist.